.coafile

```diff
@@ -40,7 +40,7 @@ ignore_length_regex = ^.*https?://
 [DOCS]
 bears = SpaceConsistencyBear, LineLengthBear
 
-files = docs/**/*.rst, CONTRIBUTING.rst, README.rst
+files = **.rst
```

Comment:
This is causing it to also find `venv/local/lib/python2.7/site-packages/setuptools-28.8.0.dist-info/DESCRIPTION.rst` in https://circleci.com/gh/coala/coala/12557

Please use `docs/**/*.rst, *.rst` for all the rst sections.